test/e2e/scheduler_predicates.go

```diff
@@ -135,6 +135,31 @@ func cleanupPods(c *client.Client, ns string) {
 	}
 }
 
+// Waits until all existing pods are scheduled and returns their amount.
+func waitForStableCluster(c *client.Client) int {
+	timeout := 10 * time.Minute
+	startTime := time.Now()
+
+	allPods, err := c.Pods(api.NamespaceAll).List(labels.Everything(), fields.Everything())
+	expectNoError(err)
+	scheduledPods, currentlyNotScheduledPods := getPodsScheduled(allPods)
+	scheduledPodsNo := len(scheduledPods)
```

Comment:
This seems unneded - please remove this variable and just return len(scheduledPods).